src/explorer/ApiManagementProvider.ts

```diff
@@ -18,6 +18,13 @@ import { getWorkspaceSetting, updateGlobalSetting } from '../vsCodeConfig/settin
 import { ServiceTreeItem } from './ServiceTreeItem';
 import { treeUtils } from '../utils/treeUtils';
 
+export function createSubscriptionTreeItem(
+    parent: AzExtParentTreeItem,
+    subscription: ISubscriptionContext,
+): AzExtTreeItem {
+    return new ApiManagementProvider(parent, subscription);
```

Comment:
can we call the constructor directly instead of creating a wrapper function?

Reply:
sure